LLama/Extensions/ListExtensions.cs

```diff
@@ -5,12 +5,14 @@ namespace LLama.Extensions
 {
     internal static class ListExtensions
     {
-#if NETSTANDARD2_0
+#if NETSTANDARD2_0 || NETSTANDARD2_1
```

Comment:
`EnsureCapacity` was introduced with `NET6` (docs: https://learn.microsoft.com/en-us/dotnet/api/system.collections.generic.list-1.ensurecapacity?view=net-7.0). So I think this can just be:

`#if !NET6_OR_GREATER`

Reply:
I've updated it to `#if !NET6_0_OR_GREATER`!